README.md

```diff
@@ -113,7 +113,7 @@ await novu.trigger('<TRIGGER_NAME>',
 
 ## Embeddable Notification Center
 
-Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
+Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an IFrame embed if you are not using React.
```

Comment:
Same comment as this previos PR with the same change proposed: https://github.com/novuhq/novu/pull/1711#discussion_r995021703